packages/core/src/scp-cf/destination-service.ts

```diff
@@ -45,7 +45,7 @@ function fetchDestinations(destinationServiceUri: string, jwt: string, type: Des
   const targetUri = `${destinationServiceUri.replace(/\/$/, '')}/destination-configuration/v1/${type}Destinations`;
 
   return callDestinationService(targetUri, jwt, options)
-    .then(response => response.data.map(d => parseDestination(d)))
+    .then(response => parseDestinations(response.data))
```

Comment:
so we actually included a breaking change by enforcing a URL to be there in the parse of the destination. Now we filter away destination returned from the service which are not type `HTTP` or `undefined`. But this means the parse does now skip these destinations. Isn`t this adding a new breaking change?

Reply:
In the past before the changes, the callDestinationService could return `RFC` destinations. Now they are filtered out.

Reply:
It might be considered as a breaking change, although it does not change the API. As an alternative I could not throw an error in those cases. Theoretically somebody could want to use an RFC destination and get a JSON object from us for that. @mr-flannery what is your take on that?

Reply:
Hmm. I do agree with Frank in that it's technically a breaking change. At the same time, do you even get a working destination for Type `RFC`, i.e. are all of the fields actually parsed correctly? If there's a way that you could potentially use these destinations with some SDK independent packages for RFC, I think it's worth keeping them (i.e. to not filter them out). Shouldn't it be enough to check the type before validating? Or maybe move the check into the validation predicate? Or is there something I'm missing here?

Reply:
What you are probably missing is a technical issue - sanitizing also checks for urls and destinations don't have a type. But I fixed that, I agree that we should not change the behavior here.